### PR TITLE
[Datastore] Fix Spark ingest to Redis with numeric index

### PR DESCRIPTION
**Pull request by alxtkr77**
[ML-3504](https://jira.iguazeng.com/browse/ML-3504)

### CODE REVIEW

File: mlrun/datastore/targets.py [hunk 1]
```diff
@@ -1189,7 +1189,7 @@ def prepare_spark_df(self, df, key_columns):
         from pyspark.sql.functions import udf
         from pyspark.sql.types import StringType
 
-        udf1 = udf(lambda x: x + "}:static", StringType())
+        udf1 = udf(lambda x: str(x) + "}:static", StringType())
```
Comment (assaf758) [inline mlrun/datastore/targets.py:1192]: what about the corresponding case of storey over redis, does it need handling as well?

**Review by assaf758 — CHANGES_REQUESTED** [review]
please add some test coverage around entity types

File: mlrun/datastore/targets.py (hunk 1 above)
Comment (alxtkr77) [reply mlrun/datastore/targets.py:1192]: No, only spark

**Comment by Tankilevitch** [comment]
please try to add link to the ticket would be easier to track :)

**Review by assaf758 — APPROVED** [review]

**Pull request merged**